inc/admin/ui/notice-backup-folder-not-writable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin\' uh?' );
+
+$this->render_view( 'header' );
+
+$filesystem  = imagify_get_filesystem();
+$backup_path = imagify_make_file_path_replative( get_imagify_backup_dir_path( true ) );
```

Comment:
Lol, you know what? It's spelled like that everywhere (it's really the name of the function) XD
I'll deprecate `imagify_make_file_path_replative()` and create `imagify_make_file_path_relative()` once merged (it's used in [some other places](https://github.com/wp-media/imagify-plugin/search?utf8=%E2%9C%93&q=imagify_make_file_path_replative&type=) in the plugin, since 1.6.8).